src/main/java/ru/geekbrains/lesson1/MainApp.java

```diff
@@ -0,0 +1,152 @@
+package ru.geekbrains.lesson1;
+
+/**
+ * Реализация домашнего задания курса Java. Уровень 1
+ *
+ * @author Kristina Retivykh
+ */
+public class MainApp {
+
+    // 1. Создать пустой проект в IntelliJ IDEA и прописать метод main().
+    public static void main(String[] args) {
+        System.out.println("\nHello, my dear World! This is task number one.");
+        createVariablres(); // task 2
+
+        System.out.println("\nThis is task number 3 aka calculate (a * (b + (c / d))).");
+        calculate(10,20.5565f,30.5f,40); // task 3
+        calculate(10,20,30.05f,0); // task 3
+
+        System.out.println("\nThis is task number 4 aka isPositiveOrNegative.");
+        System.out.println(task10and20(10,5)); //task 4
+        System.out.println(task10and20(10,50)); //task 4
+
+        System.out.println("\nThis is task number 5 aka isPositiveOrNegative.");
+        isPositiveOrNegative(4545); //task 5
+        isPositiveOrNegative(-450598); //task 5
+
+        System.out.println("\nThis is task number 6 aka isNegative.");
+        System.out.println(isNegative(4846)); //task 6
+        System.out.println(isNegative(-4846)); //task 6
+
+        System.out.println("\nThis is task number 7 aka greetUserByName.");
+        greetUserByName("William"); // task 7
+
+        System.out.println("\nThis is task number 8 aka printYearLeapStatus.");
+        printYearLeapStatus(2024); // task 8
+        printYearLeapStatus(2000); // task 8
+        printYearLeapStatus(2300); // task 8
+    }
+
+    /**
+     * Создаёт переменные всех пройденных типов данных и
+     * инициализирует их значения.
+     */
+    public static void createVariablres() {
+        byte byteValue = 127;
+        short shortValue = -32768;
+        int intValue = 1;
+        long longValue = 2550000L;
+        float floatValue = 5.25f;
+        double doubleValue = -5.26;
+        char charValue = 'r';
+        boolean boolValue = true;
+    }
+
+    /**
+     * Возвращает результат выражения a * (b + (c / d))
+     * переменные всех пройденных типов данных и
+     * инициализирует их значение
+     *
+     * @return результат вычисления
+     */
+    public static float calculate(float a, float b, float c, float d) {
+        if (d != 0)
+            return (a * (b + (c / d)));
+        System.out.println("\nYou try to divide by zero! Calculation failed, returned default value.\n");
+        return 0;
+    }
+
+    /**
+     * Проверяет, что сумма двух целых чисел лежит в пределах от 10 до 20 включительно.
+     *
+     * @return {@code true} сумма двух целых чисел лежит в пределах от 10 до 20 включительно.
+     *         {@code false} сумма выходит за пределы диапазона.
+     */
+    public static boolean task10and20(int a, int b) {
+        long sum = a + b; // long  случай суммирования больших чисел
+        if (sum >= 10 && sum <=20) {
```

Comment:
записи данного рода можно сократить до
```suggestion
        return sum >= 10 && sum <=20;
```